support-models/src/main/scala/com/gu/support/zuora/api/SubscriptionData.scala

```diff
@@ -152,25 +152,23 @@ object SubscriptionProductFeature {
 case class SubscriptionProductFeature(featureId: String)
 
 object Subscription {
-  implicit val decoder: Decoder[Subscription] = decapitalizingDecoder[Subscription].prepare(
-    _.withFocus(_.mapObject(_.renameField("PromotionCode__c", "promoCode")))
-      .withFocus(_.mapObject(_.renameField("ReaderType__c", "readerType")))
-  )
```

Comment:
this was out of date and we should never need to deserialise this as it's part of the request not the response.
I removed it completely and turned all the codecs that depend on it into Encoders and the maain code still seemed to compile okay.